soul-admin/src/test/java/org/dromara/soul/admin/service/impl/PermissionServiceImplTest.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dromara.soul.admin.service.impl;
+
+import org.dromara.soul.admin.config.JwtProperties;
+import org.dromara.soul.admin.entity.DashboardUserDO;
+import org.dromara.soul.admin.entity.PermissionDO;
+import org.dromara.soul.admin.entity.ResourceDO;
+import org.dromara.soul.admin.entity.UserRoleDO;
+import org.dromara.soul.admin.mapper.DashboardUserMapper;
+import org.dromara.soul.admin.mapper.PermissionMapper;
+import org.dromara.soul.admin.mapper.ResourceMapper;
+import org.dromara.soul.admin.mapper.UserRoleMapper;
+import org.dromara.soul.admin.spring.SpringBeanUtils;
+import org.dromara.soul.admin.vo.PermissionMenuVO;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.context.ConfigurableApplicationContext;
+
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * add test case for {@link PermissionServiceImpl}.
+ *
+ * @author HoldDie
+ * @version 1.0
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class PermissionServiceImplTest {
```

Comment:
Please keep the package name the same as elsewhere. e.g.  `org.dromara.soul.admin.service.PermissionServiceTest`